.github/CHANGELOG.md

```diff
@@ -1,11 +1,34 @@
-## Release 0.6.0a3 (development release)
+## Release 0.6.1a3 (development release)
 
 ### New features since the last release
 
 ### Bug fixes
 
 ### Improvements
 
+### Documentation changes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
+See full commit details ...
+
+---
+
+
+## Release 0.6.1a3 (current release)
```

Comment:
Should we jump multiple version numbers here, or consider multiple breaking changes as justifying a single increment to 0.5.0?

Reply:
We should have used 0.5.0 as you said based on the new rules. However, because we had some old rules and historically already used 0.6.0 for the repo just before Special Release, my only option was to go to 0.6.1. One should never go back in versioning, it's understandable to have a few bumps and unpublished versions when the project is in pre-release and rules are changing.